### PR TITLE
Adds nonprofit card

### PR DESCRIPTION
**Pull request by matthebrown**
Closes part of #369

### CODE REVIEW

File: components/core/cards/NonprofitCard.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                  void router.push(
+                    config.pages.donate(),
+                    config.pages.donate(nonprofitCardData._id)
+                  );
+                }}
+              >
+                Donate
+              </CoreButton>
+              <div className={supporters}>
+                <CoreTypography variant="h4">5 Supporters</CoreTypography>
```
Comment (matthebrown) [inline components/core/cards/NonprofitCard.tsx:177]: This will need to be modified once the nonprofits are updated with the `supporters` field.

**Comment by matthebrown** [comment]
Not sure why the CI/CD check is failing. The lint runs without issue locally.

File: components/core/cards/NonprofitCard.tsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+      <div className={cardContainer}>
+        <div className={card}>
+          <div className={header}>
+            <img className={logo} alt={`${nonprofitCardData.name}`} />
+            <div className={headerInfo}>
+              <CoreTypography variant="h2">
+                {nonprofitCardData.name}
+              </CoreTypography>
+              <CoreTypography variant="h5" className={location}>
+                Atlanta, GA
```
Comment (matthebrown) [inline components/core/cards/NonprofitCard.tsx:150]: We will also need some type of `city` or `location` field to update this properly. We could try to pull info from the nonprofit's `events`, but a nonprofit could have events in multiple cities.

Comment (PC98) [reply components/core/cards/NonprofitCard.tsx:150]: Ah good catch. I will ask Core if they are taking care of this.

File: components/core/cards/NonprofitCard.tsx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        opacity: 0.8
+      }
+    },
+    logo: {
+      width: "124px",
+      height: "124px",
```
Comment (PC98) [inline components/core/cards/NonprofitCard.tsx]: nit: just `124`

File: components/core/cards/NonprofitCard.tsx [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+      marginLeft: 24,
+      backgroundColor: "white",
```
Comment (PC98) [inline components/core/cards/NonprofitCard.tsx]: let's use `palette.background.paper` whenever you want to use "white" as a background color

File: components/core/cards/NonprofitCard.tsx [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+      textTransform: "capitalize",
+      pointerEvents: "none"
+    },
+    aboutContainer: {
+      width: 478,
+      height: 54,
+      marginTop: 20,
+      marginLeft: 24,
+      fontSize: 14,
+      lineHeight: "18.2px",
```
Comment (PC98) [inline components/core/cards/NonprofitCard.tsx:97]: might be able to drop the `px` and just a number, `18.2`

File: components/core/cards/NonprofitCard.tsx [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+              <CoreTypography variant="h2">
+                {nonprofitCardData.name}
+              </CoreTypography>
+              <CoreTypography variant="h5" className={location}>
+                Atlanta, GA
+              </CoreTypography>
+              <CoreButton variant="contained" className={cause}>
+                <CoreTypography variant="caption">
+                  {nonprofitCardData.cause.toLowerCase()}
+                </CoreTypography>
```
Comment (PC98) [inline components/core/cards/NonprofitCard.tsx]: Could you use a Chip instead: https://material-ui.com/components/chips/

File: components/core/cards/NonprofitCard.tsx [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+      zIndex: 1
+    },
+    location: {
+      marginTop: 8
+    },
+    cause: {
+      height: 34,
+      marginTop: 12,
+      background: "white",
+      color: "#333333",
```
Comment (PC98) [inline components/core/cards/NonprofitCard.tsx]: Is this gray-80? try using `palette.text.primary`
For `background: "white"`, try `backgroundColor: palette.background.paper` instead.

Though if we switch to using material-ui's Chip, you might not have to specify text color or background-color

File: utils/types.ts [hunk 8]
```diff
@@ -67,6 +67,8 @@ export type CauseCardData = {
   filterValue: string;
 };
 
+export type NonprofitCardData = Omit<Nonprofit, "stripeAccount" | "events">;
```
Comment (PC98) [inline utils/types.ts:70]: Okay I think the reason you are getting a type error is that a change went into develop branch that removed background images from all non-profits, so line 124 in `NonprofitCard.tsx` will error out. 
You can use the same background image on all non-profit cards by downloading it from Figma.

**Review by PC98 — COMMENTED** [review]
Awesome! Left some feedback but you don't have to prioritize working on it.
We will focus on getting the "Other Events Like This" list completed over the next few days.

File: components/core/cards/NonprofitCard.tsx (hunk 5 above)
Comment (matthebrown) [reply components/core/cards/NonprofitCard.tsx:97]: `lineHeight` rounds to a multiple, so we have to leave the "px"

**Review by PC98 — APPROVED** [review]

**Pull request merged**